test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -474,6 +474,24 @@ describe('Dropdown', () => {
 
       instance.setSelectedIndex.should.not.have.been.calledOnce()
     })
+
+    it('will only set the selectedIndex to the first element if forceSelection is set to true (default)', () => {
+      wrapperMount(<Dropdown options={options} />)
+      wrapper.setState({ selectedIndex: undefined })
+      wrapper.setProps({ options: [] })
+
+      wrapper.should.have.state('selectedIndex', 0)
+    })
+
+    it('will only not set the selectedIndex to the first element if forceSelection is set to false', () => {
+      wrapperMount(<Dropdown options={options} forceSelection={false} />)
+      wrapper.setProps({
+        options: [],
+      })
+
+      wrapper.setState({ selectedIndex: undefined })
+      wrapper.should.have.state('selectedIndex', undefined)
```

Comment:
```jsx
wrapper.should.have.state('selectedIndex', undefined)
```

The assertion above is broken, don't have any idea why.

Assertions below work :thinking: 
```jsx
expect(wrapper.state().selectedIndex).to.equal(undefined)
wrapper.should.have.not.state('selectedIndex')
```